src/academic/models.py

```diff
@@ -25,6 +26,31 @@ class Meta:
         verbose_name_plural = _('Procedures')
 
 
+STATUS_CHOICES = (
```

Comment:
Cuando se cambia el idioma, los valores como 'In Review'  se convierte a otro texto, por lo que no se pueden utilizar dentro de validadores como if element.status == 'In Review' xq falla.

Soluciones quitar el _() de la primera parte del STATUS_CHOICE.  
Cambiar en el templete el método para definir el estado.